test/e2e/main/main_test.go

```diff
@@ -893,7 +901,7 @@ func TestExternalServiceWithAuth(t *testing.T) {
 	tutils.ExpectNoError(err)
 
 	// Sleep for a while to be sure that the old pods are gone
-	// The restart is ongoing and it would that more than 10 sec
+	// The restart is ongoing, and it would that more than 10 sec,
```

Comment:
The comment doesn't really make sense in the first place so I think we can remove it entirely.